python/dgl/partition.py

```diff
@@ -592,5 +592,21 @@ def map_to_global(self, idxs, part_id):
             )
         )
 
+    def generate_permutation(self, idxs):
+        """Create a mapping for the given indices to different partitions,
+        and a count of the number of indices per part.
+
+        Return
+        ------
+        perm (Tensor): The permutation to re-order the indices by partition.
+        send_sum (Tensor): The number of indices per partition.
```

Comment:
Add size of each tensor in the comment will be very helpful.

Reply:
Updated.